docs/source/markdown/podman-generate-kube.1.md

```diff
@@ -19,6 +19,8 @@ Potential name conflicts between volumes are avoided by using a standard naming
 Note that if an init container is created with type `once` and the pod has been started, the init container will not show up in the generated kube YAML as `once` type init containers are deleted after they are run. If the pod has only been created and not started, it will be in the generated kube YAML.
 Init containers created with type `always` will always be generated in the kube YAML as they are never deleted, even after running to completion.
 
+Note that when using volumes and generating a Kubernetes YAML for an unprivileged and rootless podman container, you need to either add the "privileged: true" option to the pod spec or relabel the volume via the cli command "chcon -t container_file_t -R <directory>" to ensure that you have the right permissions to access the volume when the pod/container is created in a Kubernetes cluster.
```

Comment:
spc_t context would be better the privileged, also should point out that this is only on SELinux enabled systems.

Reply:
Fixed